vnext/Microsoft.ReactNative/Modules/CreateModules.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include <CreateModules.h>
+#include <Modules/WebSocketModule.h>
+#include <Modules/WebSocketModuleUwp.h>
+#include <WinRTWebSocketResource.h>
+
+// React Native
+#include <cxxreact/CxxModule.h>
+
+// Windows API
+#include <winrt/Windows.Security.Cryptography.Certificates.h>
+
+namespace Microsoft::React {
+
+std::shared_ptr<IWebSocketResource> IWebSocketResource::Make(std::string &&urlString) {
+  // TODO: use QuirkSettings
+  std::vector<winrt::Windows::Security::Cryptography::Certificates::ChainValidationResult> certExceptions;
+  return std::make_shared<WinRTWebSocketResource>(std::move(urlString), std::move(certExceptions));
+}
+
+std::unique_ptr<facebook::xplat::module::CxxModule> CreateWebSocketModule() noexcept {
+  // TODO: use QuirkSettings
+  if (false) {
+    return std::make_unique<react::uwp::WebSocketModule>();
```

Comment:
Consider renaming the old implementation to something like "LegacyWebSocketModule".

Reply:
Alright. Though, I'd rather only rename the class name and not the source or header, to keep the refactoring simple (since those two files would ultimately be delted).